library/src/main/java/com/google/maps/android/clustering/ClusterManager.java

```diff
@@ -156,6 +160,11 @@ public void clearItems() {
         }
     }
 
+    /**
+     * Adds items to clusters. After calling this method you must invoke cluster() for the state
```

Comment:
You can also make this more Javadoc friendly by linking to the `cluster()` method via `@link` (i.e. `"... you must invoke {@link #cluster()}"`). Same comment for other methods documented.

Reply:
Done in https://github.com/googlemaps/android-maps-utils/pull/627/commits/556d20c2f9655569eaa2f972410e96ec6c381619.